src/StripeController.php

```diff
@@ -233,6 +233,7 @@ public function ajaxCallback(\Payment $payment, array $form, array $form_state)
         'stripe_id' => $intent->id,
         'type' => $intent->object,
       ];
+      $payment->setStatus(new \PaymentStatusItem(STRIPE_PAYMENT_STATUS_INTENT_CREATED));
```

Comment:
This does not catch intents that come as part of a subscription. Do we want a status change for them as well? But not all subscriptions use intents (only if confirmation is required).

Reply:
My aim is that all payments get a PENDING status before the first success. I would do so for setup intents as well, yes.